packages/medusa/src/models/batch-job.ts

```diff
@@ -22,15 +25,31 @@ export class BatchJob {
   @DbAwareColumn({ type: "enum", enum: BatchJobStatus })
   status: BatchJobStatus
 
-  @Column({ type: "text", nullable: true })
+  @Column({ nullable: true })
   created_by: string | null
 
+  @ManyToOne(() => User)
+  @JoinColumn({ name: "created_by" })
+  created_by_user: User
+
   @DbAwareColumn({ type: "jsonb", nullable: true })
   context: Record<string, unknown>
 
   @DbAwareColumn({ type: "jsonb", nullable: true })
   result: Record<string, unknown>
 
+  @CreateDateColumn({ type: resolveDbType("timestamptz") })
+  processing_at: Date | null
+
+  @CreateDateColumn({ type: resolveDbType("timestamptz") })
+  awaiting_confirmation_at: Date | null
+
+  @CreateDateColumn({ type: resolveDbType("timestamptz") })
+  confirmed_at: Date | null
+
+  @CreateDateColumn({ type: resolveDbType("timestamptz") })
+  cancelled_at: Date | null
```

Comment:
```suggestion
  canceled_at: Date | null
```

Reply:
we should align the language used in the codebase, because it seems we use some English spellings and some American, and it seems we don't agree on using single or double "l" in "cancelled": https://github.com/medusajs/medusa/pull/1380#pullrequestreview-948694713. 

I think we should agree if we generally use the American or English common spelling in the codebase. This would allow us to decide what form of words to use and make arguments for certain ways of spelling. At the moment we generally use the American spellings (authorized not authorised, color not colour etc.). If we decide to use generally American spelling this should be "canceled" and not "cancelled". (and when we decide we should document the decision in the engineering handbook style guide as we have done with the writing styleguide)

Reply:
We use American English everywhere and I am pretty sure that we only have `canceled` throughout the codebase if not we should definitely fix that :D 

I am asking you to change from `cancelled` to `canceled` which is the same that Oli is asking for in the review comment you are linking to, so think we are aligned on that :)

Reply:
Ah okay, I remembered changing it and must have thought I did it in this branch too 😅 I will change it here as well